README.md

```diff
@@ -92,7 +91,7 @@ PuppetForge::User.find('puppetlabs') # => #<Forge::V3::User(/v3/users/puppetlabs
 PuppetForge::Module.all # See "Paginated Collections" below for important info about enumerating resource sets.
 
 # Find Resources with Conditions
-PuppetForge::Module.where(query: 'apache').all # See "Paginated Collections" below for important info about enumerating resource sets.
+PuppetForge::Module.where(query: 'apache') # See "Paginated Collections" below for important info about enumerating resource sets.
```

Comment:
For backwards compatibility (Yes, this is a v2.0) we should include a .all on where that returns the self of the paginated collection.